session/model/filetransfer.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package model
```

Comment:
Can you move this back? This was only done as a workaround since we can't pull `ozzo-validation` into go-lib-micro as it will break deviceauth.

Reply:
I can try :) but I did it in order to escape the import cycle dependency and keep code clean. my reasoning:
* it is a model
* it is used from limits and from session
* does not hurt to have it separately, helps with the clean definitions in limits, config and session

so:) unless you are adamant.